### ISSUE TITLE
Free-for-all deployed HumanStandardTokenFactory?

### ISSUE THREAD

**Issue by buhrmi** [ConsenSys/Tokens#90]
Hello,

I wondered if there is a deployed version of the (current version of) HumanStandardTokenFactory that is already verified on etherscan and ready to use.

Cheers,
Stefan

**Comment by simondlr** [comment]
There is yeah, but I don’t recommend using it. TokenFactory needs to update its token code. It’s not broken, just not up to date with convention (eg, using require() for example). :)

**Comment by buhrmi** [comment]
So what would need to change to get it into a "recommendable" state? :>

**Comment by simondlr** [comment]
Basically, this needs to be redeployed -> https://github.com/ConsenSys/Tokens/blob/master/contracts/HumanStandardTokenFactory.sol

**Comment by maurelian** [comment]
@simondlr so it can be redeployed as is? Is there a dapp, or address listing somewhere that would need updating?

**Comment by buhrmi** [comment]
I'm not @simondlr but yeah you can just deploy it. But I'm wondering if it can be modified to use the [ERC223 token standard](https://github.com/Dexaran/ERC223-token-standard/tree/Recommended)

**Comment by simondlr** [comment]
@maurelian the goal was to eventually move TokenFactory to use the on chain factory, but it's not atm. Just taking the local contracts and deploying them locally. Basically: if the network has a factory, use it. If not, use local files to deploy contracts.

Anyone is free to deploy a version of the factory.

@buhrmi 

ERC223 is better discussed in the Tokens Repo, rather than in Token Factory, which is the dapp to make/deploy tokens.

**Comment by manologr89** [comment]
Hi, I've deployed a token using the contract deployed at 0xA7a677183373CA33a65f00084eB81747c8AE5F03, and I'd like to know what's the source code version of the resulting HumanStandardToken.sol. Trying to validate my token at etherscan but I can't manage to make the output bytecode match with the contract one.

I've tried commits earlier but close to 24 Jun 2016, the factory deploy date with no luck.

Thanks in advance.
